### PR TITLE
Flag component

### PR DESCRIPTION
**Pull request by willeeklund**
Added updated Flag component using 'clip-path' solution

Documentation screenshot:
screen shot 2017-04-05 at 14 18 57

### CODE REVIEW

File: src/components/flag/flags.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import { camelCase } from 'lodash';
+import React from 'react'; // eslint-disable-line
```
Comment (bstream) [inline src/components/flag/flags.js]: Is this line needed here?

File: src/components/flag/flag.md [hunk 2]
```diff
@@ -9,8 +11,23 @@
...
+          <Flag size={32} countryCode={ flag } />
           <div style={{ fontSize: '1.2rem', fontFamily: '"Hack", monospace' }}>{ flag }</div>
         </div>
       ))}
     </div>
+
+Advanced usage: combined flag
+
+    <div>
+      <Flag size={64} countryCode={ 'no' } secondaryCountryCode={ 'dk' } />
```
Comment (bstream) [inline src/components/flag/flag.md]: Can replace `countryCode={ 'string' }` with `countryCode="string"`

File: src/components/flag/flags.js (hunk 1 above)
Comment (willeeklund) [inline src/components/flag/flags.js]: Not needed, removed it now.

File: src/components/flag/flag.md (hunk 2 above)
Comment (willeeklund) [inline src/components/flag/flag.md]: Sure. Fixed it now.

File: src/components/flag/flag.jsx [hunk 3]
```diff
@@ -1,47 +1,70 @@
...
+  const height = size;
+  const circleSize = size / 2.1;
+  const canadianLeftPos = -size * (3 / 16);
+
   const styles = Object.assign({
     display: 'inline-block',
-    width: `${width / 10}rem`,
-    height: `${height / 10}rem`,
+    position: 'relative',
+    width: `${size}px`,
```
Comment (bstream) [inline src/components/flag/flag.jsx]: You dont need to append `px` to size here or below, `width: size,` will evaluate to px

Comment (willeeklund) [inline src/components/flag/flag.jsx]: Ok, changed this now.

**Pull request merged**